benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -670,7 +666,7 @@ void GraphicsBenchmarkApp::ProcessKnobs()
     const bool enableSpheres = pEnableSpheres->GetValue();
     // If the LOD is empty, assuming we skipped the setup for all sphere resources at start
     // So need to do the intial setup for all resources here
-    const bool spheresAreSetUp               = !mSphereLODs.empty();
+    const bool spheresAreSetUp               = mSpheresAreSetUp;
```

Comment:
What about just removing `spheresAreSetUp` and using `mSpheresAreSetUp` in the lines below.

Reply:
The logic here need to be reworked, not just renaming, i.e. `SetupSpheresPipelines` still throw away a PipelineInterface without freeing it, I'm trying to not change much code so I trace back source PR to figure out what's the intension of existing code.

Reply:
I see